cfgov/prepaid_agreements/jinja2/prepaid_agreements/detail.html

```diff
@@ -9,6 +9,14 @@
     {{ product.name }} | Consumer Financial Protection Bureau
```

Comment:
Worth updating this while we're here?

```suggestion
    Product agreement: {{ product.name }} by {{ product.issuer_name }} | Consumer Financial Protection Bureau
```

Reply:
Hmmm, good question. I'm going to leave this as is for now since we usually try to match the `title` to the main heading on the page, but this is probably a good thing to revisit. Maybe even sitewide to make sure our `title`s are good and descriptive? I'll split that off into a separate issue.